packages/components/dropdown/src/utils.ts

```diff
@@ -93,3 +93,17 @@ export const getItemsFromChildren = (children: ReactNode): ItemsMap => {
 
   return newMap
 }
+
+export const hasChildComponent = (children: ReactNode, displayName: string): boolean => {
```

Comment:
As discussed, we may want to gather as web team, to define a common strategy about this kind of stuff... :+1: